### PR TITLE
Add Spek Alternative v0.8.2.3

### PR DESCRIPTION
**Pull request by mariobrostech**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

I'm a complete newbie at contributing to larger projects like Homebrew, so this is all very foreign to me. If someone can guide me through naming the cask according to the token reference, I'm more than happy to do so. Also, if someone is able to add (or guide me through adding) `brew cask zap` parameters to this cask, I'd really appreciate it. Thanks for your consideration!
Also, I'm not sure if the name of this package is going to be a problem with it getting approved. I did notice something on your development guide about forks with conflicting names, and I'm pretty sure this falls under that category, but I don't know what specifically needs to be changed. If you could give me a nudge in the right direction, that would be great!

### CODE REVIEW

File: Casks/spek-alternative.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask "spek-alternative" do
```
Comment (vitorgalvao) [inline Casks/spek-alternative.rb:1]: This doesn’t follow the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).

**Comment by vitorgalvao** [comment]
Thank you (and welcome), but we’ll be rejecting this. This is a fork, not popular, hasn’t received a commit in almost two years and no new releases in almost three.

**Comment by vitorgalvao** [comment]
> If someone can guide me through naming the cask according to the token reference

See [Token Overlap](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md#token-overlap) → [Forks and Apps with Conflicting Names](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#forks-and-apps-with-conflicting-names). The resulting name would be `withmorten-spek`.

> if someone is able to add (or guide me through adding) `brew cask zap` parameters to this cask, I'd really appreciate it.

Let’s leave that to your next contribution. See [`homebrew-createzap`](https://github.com/nrlquaker/homebrew-createzap) for an external project which can help.

**Comment by mariobrostech** [comment]
> Thank you (and welcome), but we’ll be rejecting this. This is a fork, not popular, hasn’t received a commit in almost two years and no new releases in almost three.
> 
> > If someone can guide me through naming the cask according to the token reference
> 
> See [Token Overlap](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md#token-overlap) → [Forks and Apps with Conflicting Names](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#forks-and-apps-with-conflicting-names). The resulting name would be `withmorten-spek`.
> 
> > if someone is able to add (or guide me through adding) `brew cask zap` parameters to this cask, I'd really appreciate it.
> 
> Let’s leave that to your next contribution. See [`homebrew-createzap`](https://github.com/nrlquaker/homebrew-createzap) for an external project which can help.

@vitorgalvao Thanks for the feedback! The only reason I tried submitting this fork is because it's the only 64-bit version of Spek (that I know of), so it properly works on macOS Catalina. The existing Spek binary on Homebrew is even more outdated than this fork, and it doesn't function properly on macOS Catalina due to it being 32-bit only. So, there's no hope of getting any variant of Spek through homebrew/cask that properly functions on newer macOS releases?

Thanks again for the tips, I will apply these to my other PRs when I get the chance!

**Comment by vitorgalvao** [comment]
> So, there's no hope of getting any variant of Spek through homebrew/cask that properly functions on newer macOS releases?

There is, if it’s more popular and has more recent releases, or if the authors of the original officially recommend a specific fork.

**Comment by mariobrostech** [comment]
> > So, there's no hope of getting any variant of Spek through homebrew/cask that properly functions on newer macOS releases?
> 
> There is, if it’s more popular and has more recent releases, or if the authors of the original officially recommend a specific fork.

@vitorgalvao There's actually an alpha version of Spek that I just found out about with some Googling. The files are on the official GitHub repo, and it appears to work on Catalina! Is there any way this can get pushed as an update (even though it's an alpha, it's the only version that actually functions properly on the newest releases of macOS, so would it belong here still?) or on one of the other Homebrew taps that support pre-release software?
[Here's the link](https://github.com/alexkay/spek/releases/download/v0.8.3/spek-0.8.4-alpha.dmg), in case you wanted to look into this.

Thanks again!

**Comment by vitorgalvao** [comment]
> Is there any way this can get pushed as an update (even though it's an alpha, it's the only version that actually functions properly on the newest releases of macOS, so would it belong here still?) or on one of the other Homebrew taps that support pre-release software?

The cask is too unpopular. It should be removed:

```
spek
30 days: n/a
90 days: n/a
365 days: 4 (#6966)
Age: 2494 days (added 2014, January 02)
```

You may wish to consider hosting your own [tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md).

**Comment by mariobrostech** [comment]
> > Is there any way this can get pushed as an update (even though it's an alpha, it's the only version that actually functions properly on the newest releases of macOS, so would it belong here still?) or on one of the other Homebrew taps that support pre-release software?
> 
> The cask is too unpopular. It should be removed:
> 
> ```
> spek
> 30 days: n/a
> 90 days: n/a
> 365 days: 4 (#6966)
> Age: 2494 days (added 2014, January 02)
> ```
> 
> You may wish to consider hosting your own [tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md).

@vitorgalvao I definitely understand why you don't view Spek as a valuable cask due to its age, but to call it too unpopular is a bit of a stretch in my mind. Spek is quite popular among audio engineers and audiophiles alike because it allows them to view the actual quality of an audio file, as opposed to just trusting the tags, which are often inaccurate. Additionally, although it's quite old, many people rely on it for a good reason: there's just no good alternative. Audacity is the closest, but it has nowhere near the same degree of control and flexibility as Spek because Audacity is more akin to a DAW than it is an audio analysis program. I completely understand and respect your decision not to host Spek on Homebrew, but I think you're mistaken in calling it unpopular.

**Comment by vitorgalvao** [comment]
> I definitely understand why you don't view Spek as a valuable cask due to its age

It’s not due to its age, it’s due to its download count, a mere 4 in the last year.

> to call it too unpopular is a bit of a stretch in my mind.

It’s not a stretch. It’s unpopular amongst our users, those are hard numbers.

> Spek is quite popular among audio engineers and audiophiles alike

So be it, but apparently those don’t use Homebrew to install Spek, which is all that matters.

**Comment by mariobrostech** [comment]
> > I definitely understand why you don't view Spek as a valuable cask due to its age
> 
> It’s not due to its age, it’s due to its download count, a mere 4 in the last year.
> 
> > to call it too unpopular is a bit of a stretch in my mind.
> 
> It’s not a stretch. It’s unpopular amongst our users, those are hard numbers.
> 
> > Spek is quite popular among audio engineers and audiophiles alike
> 
> So be it, but apparently those don’t use Homebrew to install Spek, which is all that matters.

Don't you think the fact that the package currently on Homebrew doesn't even open on OS's as old as High Sierra might have a role to play in its unpopularity within the Homebrew community? Didn't realize you were going off your numbers, sorry about the confusion.

**Comment by mariobrostech** [comment]
I just read over this thread again and realized that I'm coming across as quite aggressive in my replies. My apologies to you, @vitorgalvao - I really don't intend to come across as rude or demanding, but it's sometimes difficult for me to convey my ideas when I'm communicating solely through text. Please don't take any of my comments personally, and I'm sorry for being so harsh in my responses.

Anyways, I just wanted to let you know that I completely understand where you're coming from and respect your decision to not accept this PR.

Take care!

**Comment by vitorgalvao** [comment]
> Don't you think the fact that the package currently on Homebrew doesn't even open on OS's as old as High Sierra might have a role to play in its unpopularity within the Homebrew community?

I do. But if it truly were popular, we’d have gotten plenty more issues and/or PRs about it in the interim (that’s what happens with other apps).

> I just read over this thread again and realized that I'm coming across as quite aggressive in my replies.

Not at all. I always read replies as if they were written with a genuine smile. If they sound positive like that, that’s how I take them. That causes less confusion and conflict, and allows for more effective conversation.

**Comment by mariobrostech** [comment]
Ok, sounds good then. Thanks for the kind words and accepting my other PR!

**Pull request closed without merging**